.github/PROCESS.md

```diff
@@ -29,3 +29,45 @@ Issues and pull requests that need review. Pull requests will automatically move
 #### Done :tada:
 
 Issues and pull requests that are completed. Issues will automatically move here when they are closed. Pull requests will automatically moved here when they are merged or closed with unmerged commits.
+
+## Triaging Issues
+
+### Issues to Triage
+
+The issues that need to be triaged all have the `triage` label. In many cases the issue can be automatically processed by the Ionic Issue Bot by applying a specific label.
+
+Once another label is applied to the issue, the `triage` label is automatically be removed by the bot.
+
+### Wrong Repo
+
+If an issue does not pertain to the Ionic Framework but does pertain to another repo, it should be moved to that repo. The bot has been set up to automatically create the issue in other repositories while closing and locking the issue in this repository. Use one of the following labels to perform that action:
+
+- ionitron: cli
+- ionitron: docs
+- ionitron: stencil
+- ionitron: native
+
+### Ionic Pro Issues
+
+If the issue is associated with Ionic Pro the submitter should be told to use the [Ionic Pro Support Forum](https://ionic.zendesk.com/hc/en-us/requests/new). The issue should be closed and locked. Use the `ionitron: ionic pro` label to accomplish this.
+
+### Support Questions
+
+If the issue is a support question, the submitter should be redirected to our [forum](https://forum.ionicframework.com) or [slack channel](https://ionicworldwide.herokuapp.com/). The issue should be closed and locked. Use the `ionitron: support` label to accomplish this.
+
+### Incomplete Template
+
+If the issue template has not been filled out completely, the issue should be closed and locked. The submitter should be informed top re-submit the issue making sure they fill the form out completely. Use the `ionitron: missing template` label to accomplish this.
+
+### Issues with Open Questions
+
+In many cases, the template is mostly filled out but just missing a thing or two or you may have a question or need clarification. In such a case, the submitter should be asked to supply that information.
+
+1. create a comment requesting the additional information or clarification
+1. add the `needs reply` label to the task
```

Comment:
This should be a 2

Reply:
If you just use "1." all the time in a numbered list in MD, the MD processor will do the numbering properly so you don't have to, so if I need to insert a step in between I can just add `1. this is another step` rather than giving it some number and re-numbering everything below it.

From here: https://guides.github.com/features/mastering-markdown/#syntax

this
```
1. Item 1
1. Item 2
1. Item 3
   1. Item 3a
   1. Item 3b
```

becomes this:

1. Item 1
1. Item 2
1. Item 3
   1. Item 3a
   1. Item 3b

Reply:
Oooooo TIL